pkg/storaged/fsys-tab.jsx

```diff
@@ -380,20 +386,25 @@ export function mounting_dialog(client, block, mode, forced_options) {
             opts.push("ro");
         if (opt_never_auto)
             opts.push("x-cockpit-never-auto");
+        if (opt_nofail)
+            opts.push("nofail");
+        if (opt_netdev)
+            opts.push("netdev");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221107-142335-754c0762-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)